### DISCUSSION TITLE
Why does the dev ui use an egg icon for beans (arc dev ui)

### DISCUSSION THREAD

**Discussion by bulldog98** [quarkusio/quarkus#31979] [Q&A] [answered]
There is a [coffee bean](https://fontawesome.com/icons/coffee-bean?s=solid&f=classic) why not use that?

[a comment by an automated account is not shown]

**Comment by mkouba** [comment, marked as answer]
Because that icon was not available in the Font Awesome Free icon set of the version used at the time this Dev UI page was created. I'm not quite sure what version is used in Quarkus 3.0. @phillip-kruger knows probably more...

**Reply by bulldog98** [reply]
Ah that makes sense. As far as I know Quarkus 3.0 is still using that icon

**Reply by gsmet** [reply]
The icon is still limited to pro in recent versions so we cannot use it.
